example-workflows/README.md

```diff
@@ -0,0 +1,46 @@
+# Example Workflows for Github Actions with Google Cloud Platform
+
+These example workflows use the [setup-gcloud][action] to build and deploy applications to Google Cloud Platform.
+
+## Workflows
+
+|           Workflow              |        Description       |
+| ------------------------------- | ------------------------ |
+| [Google Kubernetes Engine](gke/)| Deploy a static site to an existing GKE cluster |
+| [Cloud Run](cloud-run/)         | Deploy a container to Cloud Run (Fully Managed)|
+
+## Before you begin
+
+1. [Create or select a Google Cloud Platform project][project].
+
+1. Make sure that billing is enabled for your Google Cloud project. [Learn how to confirm billing is enabled for your project.][billing]
+
+1. [Create a Service Account][service-account] and download a [JSON key][key].
+
+  * Base64 encode your JSON key with the following command:
+  ```
+  base64 ~/path/to/key/key.json
+  ```
+
+  * Add the encoded key to the [repository's secrets][secrets] (&#9881; Settings > Secrets), named `GOOGLE_APPLICATION_CREDENTIALS`.
+
+  * Add the service account email, `<NAME@PROJECT_ID.iam.gserviceaccount.com`, to the repository's secrets, named `SA_EMAIL`.
+
+## Resources
+
```

Comment:
Suggest re-naming section title to "GitHub Actions Resources".

Also suggest linking to their curated collection of resources rather than try to define our own: https://help.github.com/en/actions/automating-your-workflow-with-github-actions